### PR TITLE
Pd/make post

### PR DESCRIPTION
**Pull request by pnguye18**

### CODE REVIEW

**Comment by pnguye18** [comment]
@samanthahallcox Not sure if I did this pull request correctly, if you could look at it when you have the time.

File: angular/src/app/angular-material.module.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { CommonModule } from '@angular/common';
```
Comment (samanthahallcox) [inline angular/src/app/angular-material.module.ts:1]: remove angular-material.module.ts

File: angular/src/app/app.module.ts [hunk 2]
```diff
@@ -1,13 +1,19 @@
-import { NgModule } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { AngularMaterialModule } from './angular-material.module';
```
Comment (samanthahallcox) [inline angular/src/app/app.module.ts:3]: remove all Angular Material related imports

File: angular/src/app/make-post/make-post.component.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<html>
```
Comment (samanthahallcox) [inline angular/src/app/make-post/make-post.component.html:1]: remove `html`, `head`, and `body` tags

File: angular/src/app/make-post/make-post.component.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { FormGroup } from '@angular/forms';
+import { MatChipInputEvent } from '@angular/material/chips';
```
Comment (samanthahallcox) [inline angular/src/app/make-post/make-post.component.ts:4]: Remove Angular Material imports

File: angular/src/app/make-post/make-post.component.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import { COMMA, ENTER } from '@angular/cdk/keycodes';
+
+export interface Category{
```
Comment (samanthahallcox) [inline angular/src/app/make-post/make-post.component.ts:7]: should this be extracted to the Post interface?

File: angular/src/app/make-post/make-post.component.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    margin: 5%;
+    width: 70%;
+    height: 35%;
+} */
+
+.postBody{
+    width: 100%;
+}
+
+.material-icons{
```
Comment (samanthahallcox) [inline angular/src/app/make-post/make-post.component.css:12]: remove Material icon rule

File: package.json [hunk 7]
```diff
@@ -21,10 +21,13 @@
...
   "dependencies": {
     "@angular/animations": "~12.2.0",
+    "@angular/cdk": "^12.2.12",
     "@angular/common": "~12.2.0",
     "@angular/compiler": "~12.2.0",
     "@angular/core": "~12.2.0",
-    "@angular/forms": "~12.2.0",
+    "@angular/flex-layout": "^12.0.0-beta.35",
+    "@angular/forms": "^12.2.12",
+    "@angular/material": "^12.2.12",
```
Comment (samanthahallcox) [inline package.json:30]: use `npm` to remove angular material dependencies 

https://docs.npmjs.com/uninstalling-packages-and-dependencies

**Review by samanthahallcox — CHANGES_REQUESTED** [review]
See my comments above for necessary changes

File: angular/src/app/make-post/make-post.component.ts (hunk 5 above)
Comment (samanthahallcox) [reply angular/src/app/make-post/make-post.component.ts:7]: to clarify, I meant the category interface. although I'm not sure if this was just used with the chip thing

File: angular/src/app/app-routing.module.ts [hunk 8]
```diff
@@ -1,15 +1,20 @@
 import { GuidelinesComponent } from './guidelines/guidelines.component';
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+
+import { SignInComponent } from './sign-in/sign-in.component';
```
Comment (samanthahallcox) [inline angular/src/app/app-routing.module.ts:5]: remove duplicate import (SignIn)

File: angular/src/app/app-routing.module.ts [hunk 9]
```diff
@@ -1,15 +1,20 @@
...
+import { MakePostComponent } from './make-post/make-post.component';
 import { SignInComponent } from './sign-in/sign-in.component';
 import { TigerSpacesGridComponent } from './tigerspaces-grid/tigerspaces-grid.component';
 
 const routes: Routes = [
   { path: '', component: HomePageComponent },
   { path: 'home', component: HomePageComponent },
+  { path: 'signin', component: SignInComponent},
   { path: 'signup', component: SignUpFormComponent },
+  { path: 'makePost', component: MakePostComponent},
```
Comment (samanthahallcox) [inline angular/src/app/app-routing.module.ts:17]: change casing of makePost to makepost to match other route casing

File: angular/src/app/make-post/make-post.component.html [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+    <head>
+
+    </head>
+    <body>
+        <div class="row justify-content-md-center">
+            <form class="w-75">
+            <!-- <form [formGroup]="postForm">                 -->
+                <label for="title" class="form-label">Title: </label> <br>
+                <input id="title" type="text" class="form-control" formControlName="title">
+                <br>
```
Comment (samanthahallcox) [inline angular/src/app/make-post/make-post.component.html:11]: did we want a category field?

**Pull request closed without merging**